frontend/src/Portfolio.js

```
@@ -2,878 +1,148 @@
-    async function getData() {
-        set_portfolioLoading(true);
+    async function getData(ethAddress) {
+        clearData();
+        set_downloadedIcons({});
+
+        /* - Centralized main endpoint - */
+        try {
+            const rez = await fetch(`https://api.ethplorer.io/getAddressInfo/${ethAddress}?apiKey=freekey`);
+            if (rez.ok) {
+                const json = await rez.json();
+                if (json.tokens || (json && json.ETH && json.ETH.balance > 0)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)